data/rendering-apps.yml

```diff
@@ -394,19 +394,19 @@
   - finder-frontend
 - :name: service_manual_guide
   :apps:
-  - service-manual-frontend
-- :name: service_manual_homepage
+  - government-frontend
+- :name: government-frontend
   :apps:
-  - service-manual-frontend
+  - government-frontend
 - :name: service_manual_service_standard
```

Comment:
Looks like there's one missing: service_manual_homepage

https://www.gov.uk/api/content/service-manual

Reply:
Good spot! I accidentally removed it while wrestling with vscode.